receiver/awscontainerinsightreceiver/internal/k8swindows/extractors/cpu_extractor.go

```diff
@@ -0,0 +1,51 @@
+package extractors
+
+import (
+	ci "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/containerinsight"
+	awsmetrics "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/metrics"
+	cExtractor "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/cadvisor/extractors"
```

Comment:
is there a better import name than cExtractor?

Reply:
IIRC our linter will want the import names to be all lower case, i.e. `awsmetrics`

Reply:
Changed `cExtractor` to `cextractor`.
I saw import following similar pattern in cadvisor code where import was alias as `cInfo` but for now i changed it to `cextractor`

Reply:
got it, I didn't realize it as following an established pattern. If there was no pattern I would say "containerextractor" is more clear and avoids non-obvious abbreviations.  If there is an established pattern than `cExtractor` seems fine.

Approving this rev, curious what the other agents team review thinks.  This can be changed in a later PR